schema/filter_artworks.js

```diff
@@ -48,6 +48,9 @@ const FilterArtworks = {
     dimension_range: {
       type: GraphQLString,
     },
+    extra_aggregation_gene_ids: {
```

Comment:
Cool, I'm realizing now that we are going to have to parse these out from the response on the force end (thinking about https://github.com/artsy/force/issues/3953 ). Any thoughts there? Preferably this logic lives in much in metaphysics as possible, it would be nice to do one query and have these aggregations formatted in a way that is immediately useful without much fuss.